test/e2e/README.md

```diff
@@ -122,36 +124,42 @@ therefore need the following steps:
 If you need to test an updated version of Antrea, just run
 `./infra/vagrant/push_antrea.sh` and then run the tests again.
 
-By default, if a test case fails, we write some useful debug information to a
-temporary directory on disk. This information includes the detailed description
-(obtained with `kubectl describe`) and the logs (obtained with `kubectl logs`)
-of each Antrea Pod at the time the test case exited. When running the tests in
-verbose mode (i.e. with `-v`), the test logs will tell you the location of that
-temporary directory. You may also choose your own directory using
-`--logs-export-dir`. For example:
+### Running the tests with remote (existing K8s cluster)
 
-```bash
-mkdir antrea-test-logs
-go test -count=1 -v -run=TestDeletePod antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs
+If you already have a K8s cluster, these steps should be followed to run the e2e tests.
+
+First, you should provide the ssh information for each Node in the cluster. Here is an example:
+
+```text
+Host <Control-Plane-Node>
+    HostName <Control-Plane-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+Host <Worker-Node>
+    HostName <Worker-Node-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
 ```
 
-If the user provides a log directory which was used for a previous run, existing
-contents (subdirectories for each test case) will be overridden.
-By default the description and logs for Antrea Pods are only written to disk if a
-test fails. You can choose to dump this information unconditionally with
-`--logs-export-on-success`.
+Make sure the `Host` entry for each Node matches the K8s Node name. The `Port` is the port used by the ssh service on the Node.
 
-### Testing the Prometheus Integration
+Besides the configuration of the ssh connection, make sure you have added the public key to `authorized_keys` of each Node and set `PubkeyAuthentication` and `RSAAuthentication` of ssh service to `yes`.
 
-The Prometheus integration tests can be run as part of the e2e tests when
-enabled explicitly.
+Second, when running the e2e tests, kubeconfig of the cluster will be used, so make sure you have copied it to the right location, e.g. `$HOME/.kube/config` or the path specified by `-remote.kubeconfig`.
 
-* To load Antrea into the cluster with Prometheus enabled, use:
-`./infra/vagrant/push_antrea.sh --prometheus`
-* To run the Prometheus tests within the e2e suite, use:
-`go test -v antrea.io/antrea/test/e2e --prometheus`
+Third, the e2e test framework will ensure that Antrea has been deployed to the cluster by applying the deployment manifests, so the `antrea.yml` (and `antrea-windows.yml` if the cluster has Windows Nodes) should be put under the `$HOME` directory of the control-plane Node.
```

Comment:
these lines are not wrapped correctly yet

Reply:
Fixed